src/error.c

```diff
@@ -152,22 +152,46 @@ Obj FuncCURRENT_STATEMENT_LOCATION(Obj self, Obj context)
     return retlist;
 }
 
-Obj FuncPRINT_CURRENT_STATEMENT(Obj self, Obj context)
+
+Obj FuncPRINT_CURRENT_STATEMENT(Obj self, Obj stream, Obj context)
 {
-    if (context == STATE(BottomLVars))
-        return 0;
```

Comment:
Why not leave this check here at the start, *before* trying to open the output stream?

Reply:
That's a lot better. I'll adjust it.